app/src/main/java/pl/futurecollars/invoicing/App.java

```diff
@@ -12,5 +26,31 @@ public String getGreeting() {
 
   public static void main(String[] args) {
     System.out.println(new App().getGreeting());
```

Comment:
trzymając to w kodzie, za każdym razem jak bedziesz wywoływac testy itp. to bedzie się uruchamiać i dodawać ci do bazy dane, czasami się to może przydać, ale raczej rzadko wiec rekomendowałbym usunąć (jak będziesz chciała coś z tego wykorzystać to cofniesz się w historii commitów), w ostateczności zakomentować